packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -308,6 +310,17 @@ export default function PostOptionsMenu({
     },
   ];
 
+  const showSimilarPosts = useFeature(feature.similarPosts);
```

Comment:
`useConditionalFeature` with `shouldEvaluate: !!post?.id` to make sure user is enrolled only when option menu is open.

Reply:
My thinking was the context menu is only loaded on click right?
Will verify on Monday 🤔